app/Commands/Handlers/GetOWRank.php

```diff
@@ -0,0 +1,59 @@
+<?php namespace App\Commands\Handlers;
+
+use GuzzleHttp;
+use GuzzleHttp\Exception\ClientException;
+use Spatie\SlashCommand\Attachment;
+use Spatie\SlashCommand\AttachmentField;
+use Spatie\SlashCommand\Handlers\SignatureHandler;
+use Spatie\SlashCommand\Request;
+use Spatie\SlashCommand\Response;
+
+class GetOWRank extends SignatureHandler
+{
+    public $signature = 'reinbot owrank {gamertag : Xbox gamertag of the player}';
+
+    public $description = 'Get the Overwatch rank of a gamertag.';
+
+    /**
+     * Handle the given request. Remember that Slack expects a response
+     * within three seconds after the slash command was issued. If
+     * there is more time needed, dispatch a job.
+     *
+     * API Response Format:
+     * https://ow-api.herokuapp.com/docs/
+     *
+     * @param Request $request
+     *
+     * @return \Spatie\SlashCommand\Response
+     */
+    public function handle(Request $request): Response
+    {
+        $gamertag = $this->getArgument('gamertag');
+
+        try {
+            $response = (new GuzzleHttp\Client())->get("http://ow-api.herokuapp.com/profile/xbl/us/{$gamertag}");
```

Comment:
Seeing as that this API has no authentication, and doesn't need to be swapped out. This should be fin